Wikibase/Tests/Usage/InArrayUsage.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+// Should use array_key_exists
+in_array( $key, array_keys( $array ) );
+in_array( $key, array_keys );
```

Comment:
Isn’t this a false positive? `array_keys` is not a function call here.

Reply:
This is intentional. The tokenizer does not know what a function call is. All I can look for is a T_STRING token.

Reply:
Or a T_STRING token followed by a T_LPAREN or whatever the `(` token is called? (Probably with whitespace in between, and comments…)

Reply:
Sure, but why? What false-positive would that avoid? [Bareword strings](https://wiki.php.net/rfc/deprecate-bareword-strings)? ;-)

Reply:
Okay, fair enough ;)